### PR TITLE
I'm updating my "about me" md

### PR DESCRIPTION
**Pull request by hank-gov**
Completely customized the template with my personal information, including the image (grabbed from USDS website), name, organization, past orgs, hobbies and favorite website.

@KevinMHoffmanUSDS @mattcutts-gov Scan for any major misspellings or other obvious mistakes.

### CODE REVIEW

**Comment by mattcutts-gov** [comment]
I see "exercisio"

File: about-me.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://www.usds.gov/images/hank-knaack.jpg)
 
-Hi, I'm Charlene Bernard. I'm a ________ at *__________*.
+Hi, I'm Hank Knaack. I'm a designer at *USDS at DHS*.
```
Comment (KevinMHoffmanUSDS) [inline about-me.md:5]: This is true.

File: about-me.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
 - **Past Companies and Organizations:**
+East Side Strategy, Iteration Group, Hack for LA
 - **Hobbies:**
-- **Favorite Website:** []()
+food stuff, exercisio, indie movies
```
Comment (KevinMHoffmanUSDS) [inline about-me.md:10]: Also true.

File: about-me.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+- **Favorite Website:** [What the f*** should make for dinner?](http://www.whatthefuckshouldimakefordinner.com/)
```
Comment (KevinMHoffmanUSDS) [inline about-me.md:11]: I cannot verify this is true.

**Comment by mattcutts-gov** [comment]
Check in if you're okay with "exercisio"

**Comment by hank-gov** [comment]
@KevinMHoffmanUSDS do you know how to resize images?

**Comment by KevinMHoffmanUSDS** [comment]
@hank-gov I think the img tag is the only way to really go; I'll test it in mine. 

https://ourcodeworld.com/articles/read/162/tips-and-tricks-that-you-probably-don-t-know-with-the-github-markdown-in-readme-md-files

**Comment by KevinMHoffmanUSDS** [comment]
Tested it, it works: https://github.com/KevinMHoffmanUSDS/hello-world/blob/master/about-me.md

**Pull request merged**